2-ui/1-document/01-browser-environment/article.md

```diff
@@ -1,113 +1,113 @@
 }
 
-// global functions are methods of the global object:
+// গ্লোবাল ফাংশনসমূহ গ্লোবাল অবজেক্ট এর মেথড:
 window.sayHi();
 ```
 
-And here we use it as a browser window, to see the window height:
+এখানে আমরা এটিকে "browser window" হিসেবে ব্যবহার করছি, window এর height দেখতে:
 
 ```js run
 alert(window.innerHeight); // inner window height
 ```
 
-There are more window-specific methods and properties, we'll cover them later.
+window স্পেসিফিক আরো বিভিন্ন মেথড এবং প্রপার্টি আছে, পরবর্তী অধ্যায়গুলোতে আমরা এদের সম্পর্কে জানব।
 
 ## DOM (Document Object Model)
 
-Document Object Model, or DOM for short, represents all page content as objects that can be modified.
+ডকুমেন্ট অবজেক্ট মডেল (Document Object Model), বা সংক্ষেপে DOM, পেজের সকল কন্টেন্টকে অবজেক্ট হিসেবে দেখায় যা পরিবর্তন করা যায়।
 
-The `document` object is the main "entry point" to the page. We can change or create anything on the page using it.
+`document` অবজেক্ট হল পেজের প্রপার্টি এবং মেথড সমূহের অ্যাক্সেসের "entry point"। আমরা এর সাহায্যে পেজের বিভিন্ন প্রপার্টি মোডিফাই বা সংযুক্ত করতে পারি।
 
-For instance:
+যেমন:
 ```js run
-// change the background color to red
+// ব্যাকগ্রাউন্ড কালার পরিবর্তন
 document.body.style.background = "red";
 
-// change it back after 1 second
+// ১ সেকন্ড পর পূর্বের অবস্থা
 setTimeout(() => document.body.style.background = "", 1000);
 ```
 
-Here we used `document.body.style`, but there's much, much more. Properties and methods are described in the specification: [DOM Living Standard](https://dom.spec.whatwg.org).
+এখানে শুধুমাত্র `document.body.style` ব্যবহার করেছি, কিন্তু এছাড়াও আরো অনেক প্রপার্টি এবং মেথড আছে। বিস্তারিত জানতে দেখুন: [DOM Living Standard](https://dom.spec.whatwg.org).
 
-```smart header="DOM is not only for browsers"
-The DOM specification explains the structure of a document and provides objects to manipulate it. There are non-browser instruments that use DOM too.
+```smart header="DOM শুধুমাত্র ব্রাউজারের জন্য না"
+নন ব্রাউজার প্লার্টফর্ম সমূহেও DOM ব্যবহার করা যায়। DOM স্পেসিপিকেশনে অবজেক্টের স্ট্রাকচার এবং এর ম্যানিপুলেশন নিয়ে বিস্তারিত আলোচনা করা হয়। তবে ব্রাউজার ব্যতীত অন্য প্লাটফর্মের জন্যও DOM সাপোর্ট করে।
 
-For instance, server-side scripts that download HTML pages and process them can also use DOM. They may support only a part of the specification though.
+যেমন, সার্ভার সাইড স্ক্রিপ্ট HTML পেজসমুহকে ডাউনলোড করে এবং এর সাহায্যে DOM কে প্রসেস করে। তবে এরা স্পেসিপিকেশনের শুধুমাত্র কিছু প্রপার্টি বা মেথড সমর্থন করে।
 ```
 
-```smart header="CSSOM for styling"
-There's also a separate specification, [CSS Object Model (CSSOM)](https://www.w3.org/TR/cssom-1/) for CSS rules and stylesheets, that explains how they are represented as objects, and how to read and write them.
+```smart header="CSSOM হল স্ট্যাইল এর জন্য"
+এছাড়াও CSS রুলস এবং স্ট্যাইলশীট এর আলাদা স্পেসিপিকেশন আছে [CSS Object Model (CSSOM)](https://www.w3.org/TR/cssom-1/)। এখানে বর্ণনা করা হয়েছে কিভাবে CSS প্রপার্টি সমূহকে অবজেক্ট হিসেবে দেখানো হয়, এবং এদের অ্যাক্সেস করা যায়।
```

Comment:
use স্টাইলশীট instead of স্ট্যাইলশীট
and also change `স্পেসিপিকেশন` to `স্পেসিফিকেশন`